crates/solidity/testing/snapshots/bindings_output/control/emit_event/generated/0.4.11-failure.txt

```diff
@@ -0,0 +1,33 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+Parse errors:
+Error: Expected Equal or Semicolon.
+   ╭─[input.sol:7:23]
+   │
+ 7 │         emit TestEvent(x);
+   │                       ─┬─  
+   │                        ╰─── Error occurred here.
+───╯
+References and definitions: 
+   ╭─[input.sol:1:1]
+   │
+ 1 │ contract Test {
+   │          ──┬─  
+   │            ╰─── def: 1
+ 2 │     event TestEvent(int id);
+   │           ────┬────  
+   │               ╰────── def: 2
+   │ 
+ 4 │     function test_emit() public {
+   │              ────┬────  
+   │                  ╰────── def: 3
+ 5 │         int x = 1;
+   │             ┬  
+   │             ╰── def: 4
+   │ 
+ 7 │         emit TestEvent(x);
```

Comment:
Q: why `TestEvent` is regarded here as a new definition?

Reply:
It's regarded as a variable declaration `TestEvent` with type `emit`, because `emit` as a keyword was introduced in 0.4.21. The `(x)` is skipped by the parser as an error.